extra/index.js

```diff
@@ -0,0 +1,15 @@
+import { diamondShape, equilateral } from "./pattern/index";
```

Comment:
Export from '/pattern' not '/pattern/index'

Reply:
done